rust-bindings/test_proofs.txt

```diff
@@ -0,0 +1,97 @@
+05683404333717545b0a6f0c0dde9710e4d3fe2d5cc6cc0a090a0b818bab0f17, 21, df3f619804a92fdb4057192dc43dd748ea778adc52bc498ce80524c014b81119, cc128d31890be2ef01ed47503b8559caa2014b9262651db7c43ff1544acb0de1f354e87d88b0b1c87383394d52b6fc393fbacb0224665488cd31d7c6f52d4113a7d3e3c5cc5366668dd446586bc9c66fb607ea55596ce2d14db5ff964c2f7c7ffc4f5590b1313c6ce29a2b0ef6773256d3019f0a78ace7aef2efd8c2decefc317a98bd61722839c1b68d28eb38972b4b12b5912d9024e9b8d5a4cd878bc12192b1390a7d9dd6b82a, 753087568382550fdefa85d7907efd91d7ad10a6b2e4787cf97f848f4753302e
```

Comment:
do you think it makes sense to also include the script you used to generate this file? just in case we want to extend it or modify it in the future.